lib/server.js

```diff
@@ -41,6 +42,11 @@ function startServer (opts) {
     return data
   }
 
+  function resolveTopojson (url) {
+    var data = fs.readFileSync(path.join(topojsonPath, path.basename(url))).toString()
```

Comment:
yeah, that works too https://github.com/rreusser/plotly-mock-viewer/pull/12/commits/1af5824014ff1d590a01adea6913e76b2c8a2aa3